### PR TITLE
Add ability to center the content via Parameter

### PR DESCRIPTION
**Pull request by newhinton**
This adds the parameter themeWidth = "full" to center the content. this also removes the right border, which is not nessessary in this mode

screenshot_2019-02-03_16-03-18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ paginate = 5
   themeColor = "orange"
   # if you set this to 0, only submenu trigger will be visible
   showMenuItems = 2
+  # center the content on the screen
+  themeWidth = "full"
```
Comment (panr) [inline README.md]:
```suggestion
  fullWidthTheme = true // default: false
  centerTheme = true // default: false
```

File: source/css/main.css [hunk 2]
```diff
@@ -222,14 +222,22 @@ ol ol {
...
     padding: 20px;
   }
+  
 }
+.full {
+  border-right: none;
+  margin: auto;
+}
+
+.half {border-right: 1px solid rgba(255, 255, 255, .1);}
```
Comment (panr) [inline source/css/main.css]:
```suggestion

```

We don't need this ;-)

File: layouts/_default/baseof.html [hunk 3]
```diff
@@ -7,7 +7,12 @@
   {{ partial "head.html" . }}
 </head>
 <body class="">
-<div class="container">
+{{ if eq $.Site.Params.themeWidth "full" }}
+     <div class="container full">
+{{ else }}
+    <div class="container half">
+{{ end }}
```
Comment (panr) [inline layouts/_default/baseof.html]: I would prefer this:

```
{{ $container := cond $.Site.Params.FullWidthTheme "container full" (cond $.Site.Params.CenterTheme "container center" "container") }}

<div class="{{- $container -}}">
```

I know it's soooooo verbose, but I think it's better than (error prone spaces):

```
{{ $themeSize := cond $.Site.Params.FullWidthTheme " full" (cond $.Site.Params.CenterTheme "  center" "") }}

<div class="container {{- $themeSize}}">
```

File: source/css/main.css [hunk 4]
```diff
@@ -222,14 +222,22 @@ ol ol {
...
 
   @media (--phone) {
     padding: 20px;
   }
+  
 }
+.full {
+  border-right: none;
+  margin: auto;
+}
```
Comment (panr) [inline source/css/main.css]: We should move it to `container` scope like this:

```
.container {
  {...}

  &.full,
  &.center {
    border: none;
    margin: 0 auto;
  }

  &.full {
    max-width: 100%;
  }
}
```

**Comment by panr** [comment]
@newhinton do you still want to work on this? 😉

**Comment by newhinton** [comment]
@panr yes! just hadn't the time to update my pr :D

**Comment by newhinton** [comment]
I think i'll have time later this day :D

File: source/css/main.css (hunk 2 above)
Comment (newhinton) [reply source/css/main.css]: Removed :D

**Comment by panr** [comment]
@newhinton Sh... Something went wrong... You deleted some of the core js code 🤔 

Do you mind, if I close this PR and make this feature on my own? Or would you like to rebuild theme, resolve conflicts and make sure that everything is on its place? 😉 

Let me know 👌

**Comment by newhinton** [comment]
uh yeah you are right, i did delete some files

i'll check it later today, if i can rectify this easily i will update this, if not i'll tell you ;)

**Comment by newhinton** [comment]
@panr should be fixed now

There are some static files i accidentally commited in my first commit, but that shouldn't be that bad

**Comment by panr** [comment]
@newhinton conflicts 😛

**Comment by newhinton** [comment]
I hope this didn't break anything, but since those are only static files that are beeing regenerated on a build it should be fine (hopefully :D :D )

**Comment by panr** [comment]
Done! Thanks! 🎉

**Pull request merged**